buildSrc/build.gradle

```diff
@@ -20,6 +22,7 @@ dependencies {
         exclude group: 'com.github.jknack', module: 'handlebars'
     }
 
+    // We can't rely on the dependency-managed version of lombok here, so we have to hardcode it:
```

Comment:
why not?

Reply:
The `buildSrc` special project is built before the main project, and so it can't get any configuration from the main project. I'll update the comment to make that clearer.